packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -171,4 +172,26 @@ export interface TreeViewEvents {
 	 * This does NOT include changes to the content (fields/children) of the root node: for that case subscribe to events on the root node.
 	 */
 	rootChanged(): void;
+
+	/**
+	 * Fired when a revertible made on this view is disposed.
+	 *
+	 * @param revertible - The revertible that was disposed.
+	 */
+	revertibleDisposed(revertible: Revertible): void;
+
+	/**
+	 * Fired when:
```

Comment:
inherit doc should work, it just incorrectly reports `(undocumented) in the API report since that considers all APIs whos don't only contain tags to be undocumented. Thats a known API extractor bug in the reporting. Josh can confirm with more confidence though.

Reply:
That bug has actually been fixed, or at least it should have been. If you still see `{@inheritDoc}`-only comments resulting in `// (undocumented)`, I would be interested to see them.

Reply:
The only downside to `{@inheritDoc}` is that intellisense doesn't know how to resolve it, so it will show the string literal `{@inheritDoc Foo}`. But you can at least then navigate to the target to view the docs. It's a trade-off, but probably better than duplication?